src/invidious/views/community.ecr

```diff
@@ -20,6 +20,9 @@
         <div class="channel-profile">
             <img src="/ggpht<%= URI.parse(channel.author_thumbnail).request_target %>">
             <span><%= author %></span>
+            <% if channel.verified %>
+                <i class="icon ion ion-md-checkmark-circle"></i>
+            <% end %>
```

Comment:
Note that the content can be inlined with the `if` statement like so:
```
<% if channel.verified %><i class="icon ion ion-md-checkmark-circle"></i><% end %>
```

This reduces the amount of characters sent to the client (the indentations in this file are not removed).

Reply:
Okay thank you, as mentioned in the issue this is my first time using crystal language.